### DISCUSSION TITLE
v.0.11.0

### DISCUSSION THREAD

**Discussion by DavidLazarescu** [Librum-Reader/Librum#92] [Announcements]
Beta-Release

- Added book specific color-theme saving
- Mutli-book selection
- Book duplication warning
- Chinese translation
- Bug fixes

<hr /><em>This discussion was created from the release <a href='https://github.com/Librum-Reader/Librum/releases/tag/v.0.11.0'>v.0.11.0</a>.</em>

**Comment by Furshia** [comment]
Awesome, I stumbled upon your comment from reddit that was 2 years ago and glad that it has been released already now 

Thank you! Keep up the Good Work!

**Comment by petruut** [comment]
Is there any easily understandable docker-compose.yml available? I love your ambition with this, and would love to deploy it to be part of the journey.

**Reply by DavidLazarescu** [reply]
Hey, check out the Librum-Server depo. It contains a docker compose file.
